src/Datadog.Sketches/Datadog.Sketches.csproj

```diff
@@ -3,5 +3,4 @@
   <PropertyGroup>
     <TargetFrameworks>net461;netstandard2.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
Opinionated, but as this is a new library, can we enable the following:

- Nullable reference types
- Namespace declarations (instead of blocks)
- Global usings (optional, I know you're less convinced on this one, but I don't think `using System` etc adds a lot 😉 )

Reply:
Agreed. I wanted to but I forgot. Gonna do that in the next PR